### ISSUE TITLE
[question] Is there a way to get msg from errors.Wrap(code, msg)?

### ISSUE THREAD

**Issue by allencloud** [pkg/errors#164]
Hi, I have a question for the project.

Now there is a line of code : 
```
func A() error{
     return errors.Wrap(code, msg)
}
```

Then I have a caller `err := A()`, so can I use the returned `err` to get `msg`?

Thanks in advance.

**Comment by davecheney** [comment]
Not really. This was a deliberate choice to avoid using the string form of an error to transmit context between locations in the call stack

**Comment by gumingqi** [comment]
I want to set error code,and use it in the caller too
